### PR TITLE
Add '/interface ovpn-client' path

### LINKED ISSUE #242: OVPN client path not available
##### SUMMARY
Please add support for ovpn-client interfaces as well.

### PR DESCRIPTION
**Pull request by derdeagle**
##### SUMMARY
Fixes #242 - add "/interface ovpn-client" path.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
- api_info
- api_modify

##### ADDITIONAL INFORMATION
None.

### CODE REVIEW

File: changelogs/fragments/244-add-interface-ovpn-client-path.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - api_info, api_modify - add ``/interface ovpn-client`` path (https://github.com/ansible-collections/community.routeros/issues/242, https://github.com/ansible-collections/community.routeros/pull/244).
```
Comment (felixfontein) [inline changelogs/fragments/244-add-interface-ovpn-client-path.yml:2]:
```suggestion
  - api_info, api_modify - add ``interface ovpn-client`` path (https://github.com/ansible-collections/community.routeros/issues/242, https://github.com/ansible-collections/community.routeros/pull/244).
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution!

**Comment by derdeagle** [comment]
> Thanks for your contribution!

Always a pleasure ;)

**Comment by felixfontein** [comment]
@derdeagle thanks! :)

**Pull request merged**